PowerShell/ScubaGear/baselines/defender.md

```diff
@@ -2,45 +2,19 @@
 
```

Comment:
For [policy group 1](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/defender.md#license-requirements), need to add the "However, third-party solutions can be used to meet this requirement." (or similar language as you see fit) to indicate 3rd party service usage.

"Defender for Office 365 capabilities require Defender for Office 365 Plan 1 or 2. These are included with E5 and G5 and are available as add-ons for E3 and G3. However, third-party solutions can be used to meet this requirement."

Reply:
added!